### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# 1.9.0 2021-06-28
+
+* Provides new ETL routine written in Java, that will replace all Jython scripts at some point [PR](https://github.com/qbicsoftware/etl-scripts/pull/85)
+* Support for nf-core pipeline result registration [PR](https://github.com/qbicsoftware/etl-scripts/pull/85)
+* Provides metadata validation for imaging data (OMERO etl). [PR](https://github.com/qbicsoftware/etl-scripts/pull/83)
```
Comment (KochTobi) [inline CHANGELOG.md]:
```suggestion
* Provides new ETL routine written in Java, that will replace all Jython scripts at some point [(#85)](https://github.com/qbicsoftware/etl-scripts/pull/85)
* Support for nf-core pipeline result registration [(#85)](https://github.com/qbicsoftware/etl-scripts/pull/85)
* Provides metadata validation for imaging data (OMERO etl). [(#85)](https://github.com/qbicsoftware/etl-scripts/pull/83)
```

I think it makes sense to stay consistent with other repositories concerning the style of the changelog. Apart from that this looks very nice :+1:

File: drop-boxes/register-all-dropbox/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# New ETL logic written in Java
+
+Please find the source code of the ETL routine that this article is referring to in the
+[Java openBIS dropboxes](https://github.com/qbicsoftware/java-openbis-dropboxes) Github repository.
+
+## Installation
+
+Please provide the Java binaries as JAR from the [Java openBIS dropbox](https://github.com/qbicsoftware/java-openbis-dropboxes) in this directories
+folder `./lib`.
```
Comment (KochTobi) [inline drop-boxes/register-all-dropbox/README.md]:
```suggestion
Please provide the Java binaries as JAR from the [Java openBIS dropbox](https://github.com/qbicsoftware/java-openbis-dropboxes) in this repositories
folder `./lib`.
```

File: drop-boxes/register-all-dropbox/plugin.properties [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+# Drop box for registering a fastq file as a data set
+#
+# Variables:
+#   incoming-root-dir
+#     Path to the directory which contains incoming directories for drop boxes.
+incoming-dir = ${incoming-root-dir}/QBiC-register-all-data
+incoming-data-completeness-condition = marker-file
+top-level-data-set-handler = ch.systemsx.cisd.etlserver.registrator.api.v2.JavaTopLevelDataSetHandlerV2
+program-class = life.qbic.registration.MainETL
+storage-processor = ch.systemsx.cisd.etlserver.DefaultStorageProcessor
```
Comment (KochTobi) [inline drop-boxes/register-all-dropbox/plugin.properties]: Does it make sense to separate the variable parts from the invariable ones? E.g.
```suggestion

incoming-data-completeness-condition = marker-file
top-level-data-set-handler = ch.systemsx.cisd.etlserver.registrator.api.v2.JavaTopLevelDataSetHandlerV2
program-class = life.qbic.registration.MainETL
storage-processor = ch.systemsx.cisd.etlserver.DefaultStorageProcessor
# Variables:
#   incoming-root-dir
#     Path to the directory which contains incoming directories for drop boxes.
incoming-dir = ${incoming-root-dir}/QBiC-register-all-data
```

File: drop-boxes/register-omero-metadata/register-omero.py [hunk 4]
```diff
@@ -58,7 +59,8 @@
 INCOMING_DATE_FORMAT = '%d.%m.%Y'
 OPENBIS_DATE_FORMAT = '%Y-%m-%d'
 
-PROPPERTY_FILTER_LIST = ["IMAGE_FILE_NAME", "INSTRUMENT_USER", "IMAGING_DATE"]
+PROPPERTY_FILTER_LIST = ["IMAGE_FILENAME", "INSTRUMENT_USER", "IMAGING_DATE"]
+
```
Comment (KochTobi) [inline drop-boxes/register-omero-metadata/register-omero.py:63]:
```suggestion
```
nit

**Review by KochTobi — DISMISSED** [review]
LGTM. Some suggestions to the changelog but otherwise this looks very nice.

File: drop-boxes/register-all-dropbox/plugin.properties (hunk 3 above)
Comment (sven1103) [reply drop-boxes/register-all-dropbox/plugin.properties]: Sure, i just sticked to the default layout in the other dropboxes

**Review by KochTobi — APPROVED** [review]

**Comment by sven1103** [comment]
thanks a lot @KochTobi

**Pull request merged**